pkgs/ffigen/lib/src/config_provider/spec_utils.dart

```diff
@@ -309,6 +309,17 @@ Headers headersExtractor(
 /// error and throws an Exception if not found.
 String findDylibAtDefaultLocations() {
   String? k;
+  final condaEnvPath = Platform.environment['CONDA_PREFIX'] ?? '';
```

Comment:
I'm going to assume that no-one who uses Conda would like to not use their libclang from not-Conda. So I think it's fine to put Conda before the other resolution paths.